app/views/layouts/course.html.slim

```diff
@@ -9,9 +9,8 @@
             = display_course_logo(current_course)
         - if current_course_user.present? && can?(:participate, current_course)
           div.col-xs-12.user
-            table
```

Comment:
I added it in, in the earlier commit #1439 as the span were displaying as table-cell.
Turns out this "table" is redundant so I removed it in this commit.